src/components/AddFarm/AddFarm.js

```diff
@@ -0,0 +1,169 @@
+import {
+  TextField, Button,
+} from '@material-ui/core';
+import React from 'react';
+import './AddFarm.css';
+
+// Airtable set-up here
+const Airtable = require('airtable');
+
+const base = new Airtable({ apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY })
+  .base(process.env.REACT_APP_AIRTABLE_BASE_KEY);
+
+export default class AddFarm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      farmName: null,
+      description: null,
+      /* userID: null, */
+      email: null,
+      phone: null,
+      address: null,
+      zipCode: null,
+      error: null,
+      // showAlert: false,
+    };
+
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.checkFormInput = this.checkFormInput.bind(this);
+  }
+
+  /* Handles when the form is submitted, passes info to airtable */
+  handleSubmit(event) {
+    event.preventDefault();
+    const {
+      farmName, description, email, phone, address, zipCode, error,
+    } = this.state;
+
+    if (error === 'No error') {
+      base('Farms').create([
+        {
+          fields: {
+            'farm name': farmName,
+            description,
+            email,
+            phone,
+            address,
+            'zip code': Number(zipCode),
+          },
+        },
+      ], (err) => {
+        if (err) {
+          this.setState({ error: error.length === 0 ? err : `${error},  ${err}` });
+        }
+      });
+
+      this.setState({ // Reset state to initial blank values
+        farmName: null,
+        description: null,
+        // userID: null,
+        email: null,
+        phone: null,
+        address: null,
+        zipCode: null,
+        error: null,
+        // showAlert: false,
```

Comment:
Same as before, take out showAlert if you aren't using it, or consider incorporating Alert from material-ui/lab